spec/features/admins/admin_deletes_a_users_playlist_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+feature 'Admin deletes a users playlist', %q(
+As an admin
+I want to be able to delete a playlist
+So that I can monitor explicit material
+
+Acceptance Criteria
+[x] I must be able to delete a specific playlist
+[ ] Any reviews associated with the playlist must be deleted
+[ ] The owner of the playlist will be notified by email when a playlist is deleted
+
+) do
+
+  scenario "Admin deletes a users playlist" do
+    admin_user = FactoryGirl.create(:user, admin: true)
+    non_admin_user = FactoryGirl.create(:playlist)
+
+    sign_in_as(admin_user)
+    visit playlist_path(non_admin_user)
+
+    expect(page).to have_content 'Delete Playlist'
```

Comment:
Same comment as above about not needing multiple expecting content methods between user behavior.